[2025-02-05T12:19:34.476+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-02-05T12:19:34.487+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.dim_vendedor_seed manual__2025-02-05T12:19:26+00:00 [queued]>
[2025-02-05T12:19:34.493+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_dbt-model_sales.model_sales.dim_vendedor_seed manual__2025-02-05T12:19:26+00:00 [queued]>
[2025-02-05T12:19:34.494+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-02-05T12:19:34.503+0000] {taskinstance.py:2330} INFO - Executing <Task(DbtSeedLocalOperator): model_sales.dim_vendedor_seed> on 2025-02-05 12:19:26+00:00
[2025-02-05T12:19:34.507+0000] {standard_task_runner.py:63} INFO - Started process 2869 to run task
[2025-02-05T12:19:34.509+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'dag_dbt-model_sales', 'model_sales.dim_vendedor_seed', 'manual__2025-02-05T12:19:26+00:00', '--job-id', '86', '--raw', '--subdir', 'DAGS_FOLDER/dag_dbt-model_sales.py', '--cfg-path', '/tmp/tmpt9xxar4e']
[2025-02-05T12:19:34.510+0000] {standard_task_runner.py:91} INFO - Job 86: Subtask model_sales.dim_vendedor_seed
[2025-02-05T12:19:34.554+0000] {task_command.py:426} INFO - Running <TaskInstance: dag_dbt-model_sales.model_sales.dim_vendedor_seed manual__2025-02-05T12:19:26+00:00 [running]> on host 98025f4a542d
[2025-02-05T12:19:34.617+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_dbt-model_sales' AIRFLOW_CTX_TASK_ID='model_sales.dim_vendedor_seed' AIRFLOW_CTX_EXECUTION_DATE='2025-02-05T12:19:26+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-05T12:19:26+00:00'
[2025-02-05T12:19:34.619+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-02-05T12:19:36.141+0000] {local.py:220} INFO - dbtRunner is available. Using dbtRunner for invoking dbt.
[2025-02-05T12:19:36.143+0000] {local.py:436} INFO - Cloning project to writable temp directory /tmp/tmpd_gy595x from /usr/local/airflow/dags/dbt/localiza_dw
[2025-02-05T12:19:36.150+0000] {local.py:447} INFO - Partial parse is enabled and the latest partial parse file is /tmp/cosmos/dag_dbt-model_sales__model_sales/target/partial_parse.msgpack
[2025-02-05T12:19:36.154+0000] {config.py:348} INFO - Profile caching is enable.
[2025-02-05T12:19:36.161+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2025-02-05T12:19:36.162+0000] {config.py:328} INFO - Profile found in cache using profile: /tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf/profiles.yml.
[2025-02-05T12:19:36.163+0000] {local.py:408} INFO - Trying to run dbtRunner with:
 ['seed', '--vars', '{"PROD_1": P003,                                 "PROD_2": "",                                 "DATA_INICIO": 1999-01-01,                                 "DATA_FIM": 2025-02-05}', '--models', 'dim_vendedor', '--project-dir', '/tmp/tmpd_gy595x', '--profiles-dir', '/tmp/cosmos/profile/47873256e3df2d6067f929148b3671caee72f33396318ef25aae33418849eddf', '--profile', '***', '--target', 'prod']
 in /tmp/tmpd_gy595x
[2025-02-05T12:19:36.177+0000] {logging_mixin.py:188} INFO - 12:19:36  Running with dbt=1.6.1
[2025-02-05T12:19:36.261+0000] {logging_mixin.py:188} INFO - 12:19:36  Registered adapter: postgres=1.6.1
[2025-02-05T12:19:36.282+0000] {logging_mixin.py:188} INFO - 12:19:36  Unable to do partial parsing because config vars, config profile, or config target have changed
[2025-02-05T12:19:37.002+0000] {logging_mixin.py:188} INFO - 12:19:37  Found 4 models, 1 snapshot, 4 seeds, 5 tests, 3 sources, 0 exposures, 0 metrics, 349 macros, 0 groups, 0 semantic models
[2025-02-05T12:19:37.004+0000] {logging_mixin.py:188} INFO - 12:19:37
[2025-02-05T12:19:37.059+0000] {logging_mixin.py:188} INFO - 12:19:37  Concurrency: 1 threads (target='prod')
[2025-02-05T12:19:37.059+0000] {logging_mixin.py:188} INFO - 12:19:37
[2025-02-05T12:19:37.062+0000] {logging_mixin.py:188} INFO - 12:19:37  1 of 1 START seed file dm_sales.dim_vendedor ................................... [RUN]
[2025-02-05T12:19:37.129+0000] {logging_mixin.py:188} INFO - 12:19:37  1 of 1 OK loaded seed file dm_sales.dim_vendedor ............................... [INSERT 20 in 0.06s]
[2025-02-05T12:19:37.137+0000] {logging_mixin.py:188} INFO - 12:19:37
[2025-02-05T12:19:37.139+0000] {logging_mixin.py:188} INFO - 12:19:37  Finished running 1 seed in 0 hours 0 minutes and 0.13 seconds (0.13s).
[2025-02-05T12:19:37.146+0000] {logging_mixin.py:188} INFO - 12:19:37
[2025-02-05T12:19:37.147+0000] {logging_mixin.py:188} INFO - 12:19:37  Completed successfully
[2025-02-05T12:19:37.148+0000] {logging_mixin.py:188} INFO - 12:19:37
[2025-02-05T12:19:37.149+0000] {logging_mixin.py:188} INFO - 12:19:37  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
[2025-02-05T12:19:38.417+0000] {local.py:138} WARNING - Artifact schema version: https://schemas.getdbt.com/dbt/manifest/v10.json is above dbt-ol supported version 7. This might cause errors.
[2025-02-05T12:19:38.419+0000] {local.py:494} INFO - Inlets: []
[2025-02-05T12:19:38.420+0000] {local.py:495} INFO - Outlets: []
[2025-02-05T12:19:38.420+0000] {local.py:595} INFO - Assigning inlets/outlets without DatasetAlias
[2025-02-05T12:19:38.421+0000] {dag.py:3096} INFO - Sync 1 DAGs
[2025-02-05T12:19:38.430+0000] {dag.py:3954} INFO - Setting next_dagrun for dag_dbt-model_sales to None, run_after=None
[2025-02-05T12:19:38.460+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-02-05T12:19:38.467+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=dag_dbt-model_sales, task_id=model_sales.dim_vendedor_seed, run_id=manual__2025-02-05T12:19:26+00:00, execution_date=20250205T121926, start_date=20250205T121934, end_date=20250205T121938
[2025-02-05T12:19:38.483+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-02-05T12:19:38.501+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-05T12:19:38.503+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
